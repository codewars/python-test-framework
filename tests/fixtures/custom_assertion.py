import codewars_test as test


def custom_assert_equal(a, b):
    if a != b:
        raise AssertionError("Expected {} to equal {}".format(a, b))


@test.describe("group 1")
def group_1():
    @test.it("test 1")
    def test_1():
        custom_assert_equal(1, 1)

    @test.it("test 2")
    def test_2():
        custom_assert_equal(1, 2)

    @test.it("test 3")
    def test_3():
        assert 1 == 2, "using assert"
