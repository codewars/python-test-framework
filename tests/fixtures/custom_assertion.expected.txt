
<DESCRIBE::>group 1

<IT::>test 1

<PASSED::>Test Passed

<COMPLETEDIN::>0.00

<IT::>test 2

<FAILED::>Expected 1 to equal 2

<COMPLETEDIN::>0.01

<IT::>test 3

<FAILED::>using assert

<COMPLETEDIN::>0.00

<COMPLETEDIN::>0.03
